packages/babel-plugin-proposal-class-properties/test/fixtures/static-property-tdz/edgest-case/expected.js

```diff
@@ -1,17 +0,0 @@
-function _classCallCheck(instance, Constructor) { if (!(instance instanceof Constructor)) { throw new TypeError("Cannot call a class as a function"); } }
-
-function _classNameTDZError(name) { throw new Error("Class \"" + name + "\" cannot be referenced in computed property keys."); }
-
-let A = function A() {
-  _classCallCheck(this, A);
-};
-
-var _x$x = {
-  x: (_classNameTDZError("A"), A) || 0
-}.x;
-Object.defineProperty(A, _x$x, {
-  configurable: true,
-  enumerable: true,
-  writable: true,
-  value: void 0
-});
```

Comment:
All these `expected.js` in `static-property-tdz` shouldn't exist because the transform is supposed to throw.

Reply:
I accidentally committed these two files almost three times lol

Reply:
Looking at the spec, it seems that the error is supposed to happen at runtime, since it's not specified as an early error anywhere. But that's what the plugin does anyway, so probably better to file that as a separate bug 🤷‍♀️